### DISCUSSION TITLE
Gamecube Game Box Art

### DISCUSSION THREAD

**Discussion by wgmcgovern** [emukidid/swiss-gc#592] [Q&A]
Is there a process to add gamecube game box art to view in Swiss?

**Comment by emukidid** [comment]
There isn't, but I plan to add it in at some point as an optional feature.

**Reply by TheBauer99** [reply]
I would love to see that.

**Reply by MrRussellgro** [reply]
> I would love to see that.

Same, dude! It would be useful for my collection I have.

**Reply by Chuckles1001** [reply]
Has there been any movement on this with Box art, cart art or video wheel? Thanks
